.circleci/config.yml

```diff
@@ -133,12 +133,20 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/epoch-latest-ubuntu-x86_64.tar.gz
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key `basename ${PACKAGE_TARBALL:?}
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key epoch-latest-ubuntu-x86_64.tar.gz
 
   upload_release_artifacts: &upload_release_artifacts
     run:
       name: Upload release artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/epoch-latest-ubuntu-x86_64.tar.gz
```

Comment:
Hardcoding the arch is not very future proof, what if we decide to publish the OSX packages (with the same CircleCI task) ?

Reply:
Could you please use `PKG_SUFFIX` all over the commands, and please also make it required by `${PKG_SUFFIX:?}`